packages/offix-cache/src/cache/mutations.ts

```diff
@@ -5,9 +5,15 @@ import { Query } from "./CacheUpdates";
 import { getOperationFieldName, deconstructQuery } from "../utils/helperFunctions";
 import { isArray } from "util";
 
+export interface Context {
+  conflictBase?: any;
+  returnType?: string;
```

Comment:
I do not understand this.
We have explicit fields added bellow (idField and itType).
Is the fact that we attaching those to context internal implementation specific.
Missing comment with motivation for this interface